http/src/request/application/command/create_global_command/chat_input.rs

```diff
@@ -121,6 +128,17 @@ impl<'a> CreateGlobalChatInputCommand<'a> {
     /// Set the localization dictionary for the command name.
     ///
     /// Defaults to [`None`].
+    ///
+    /// # Errors
+    ///
+    /// Returns an error of type [`NameLengthInvalid`] if the length is invalid.
+    ///
+    /// Returns an error of type [`NameCharacterInvalid`] if the name contains a
+    /// non-alphanumeric character or an uppercase character for which a
+    /// lowercase variant exists.
```

Comment:
Could this just link to [the docs for `chat_input_name`](https://docs.rs/twilight-validate/latest/twilight_validate/command/fn.chat_input_name.html) to there is less to keep in sync if that documentation improves?

Reply:
i think this might fall into the potential followup, itd be better to get all these done at once